suite-native/module-assets/src/devicesSlice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
This file shouldn't be in this package. I think we should create devices package here in suite-native, even if it will be only temporary